tests/includes/core/classes/class-test-query.php

```diff
@@ -120,4 +120,31 @@ public function test_get_past_events(): void {
 		$this->assertSame( 'gp_event', $response->query['post_type'], 'Failed to assert post type is gp_event.' );
 	}
 
+	/**
+	 * Coverage for get_past_events method.
+	 *
+	 * @covers ::adjust_admin_event_sorting
+	 *
+	 * @return void
+	 */
+	public function test_adjust_admin_event_sorting(): void {
+		$instance = Query::get_instance();
+
+		$this->mock->user( false, 'admin' );
+		$response = $instance->adjust_admin_event_sorting( array() );
+		$this->assertEmpty( $response, 'The array is not empty.' );
```

Comment:
I've been following a convention I picked up from PMC where we write message as `Failed to assert array is not empty`